sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_layout.py

```diff
@@ -0,0 +1,118 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_layout.py
+
+DESCRIPTION:
+    This sample demonstrates how to extract text, selection marks, and layout information from a document
```

Comment:
I dont recall if the service still returns all of these elements in the layout model anymore. Worth taking a look to make sure later.